Tests/SonarScanner.MSBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -267,6 +261,127 @@ public void PreProc_EndToEnd_SuccessCase_With_Organization()
             AssertAnalysisConfig(settings.AnalysisConfigFilePath, 2, logger);
         }
 
+        [DataTestMethod]
+        [DataRow("6.7.0.22152")]
+        [DataRow("7.0.0.22152")]
```

Comment:
As the version-specific logic is well tested in `SonarWebServiceTest`, I'd say that we should be fine with a single value in this test. We just want to make sure we invoke it.